test/flows.spec.js

```diff
@@ -380,12 +371,7 @@ describe('Flows', function() {
         stripe.charges.create({
           amount: 1234,
           currency: CURRENCY,
-          card: {
-            number: '4000000000000002',
-            exp_month: 12,
-            exp_year: exp_year,
-            cvc: 123,
-          },
+          card: 'tok_chargeDeclined',
```

Comment:
Wow, about a million times more readable if you don't know about magic card numbers. +1.